mitodl/mitxonline
# feat!: remove `Course.position_in_program`

#### Pre-Flight checklist


#### What are the relevant tickets?
The first part of https://github.com/mitodl/mitxonline/issues/1220

#### What's this PR do?
Removes `Course.position_in_program`. We will be removing the `Course.program` in the follow-up PRs.

#### How should this be manually tested?

- Verify that the `Course.position_in_program` is removed.
- Verify that everything is working fine.
- Verify that the properties/methods removed are not used anywhere and are safely removed.

#### Any background context you want to provide?
As per the original ticket, it would be hard to manage the changes in a single PR. After a discussion, we decided to create multiple smaller PRs having similar kinds of changes. This is the first PR and it only removes changes related to `Course.position_in_program`.
